examples/realtime_example/main.cpp

```diff
@@ -94,55 +116,6 @@ void render_drawables() {
   }
 }
 
-// 
-// Dart accessible funcitons
-// These need to be exposed as "C" functions and be exported
-//
-#if defined(_WIN32)
```

Comment:
I think you can better read the Code when not all Functions in one File.

main Builds the app and run it.
exportfunc all Functions are Exported to Dart VM to used from there

Reply:
I partially agree, but for now I'd like to leave it as is. It will also help keep the PR more focused on one feature -- adding / fixing build functionality.

If you want to move the exported functions, you can open a separate PR for it.